devicetypes/smartthings/zwave-switch-generic.src/zwave-switch-generic.groovy

```diff
@@ -14,12 +14,15 @@
 metadata {
 	definition (name: "Z-Wave Switch Generic", namespace: "smartthings", author: "SmartThings") {
 		capability "Actuator"
+		capability "Health Check"
  		capability "Switch"
 		capability "Polling"
 		capability "Refresh"
 		capability "Sensor"
 
 		fingerprint inClusters: "0x25", deviceJoinName: "Z-Wave Switch"
+		fingerprint mfr:"001D", prod:"1A02", model:"0334", deviceJoinName: "Leviton DZPA1 Plug-in Appliance Module"
```

Comment:
Actually maybe it would be better to split the fingerprint additions into another PR to simplify things, because there are a few Leviton devices we need to add for cert: 
https://smartthings.atlassian.net/browse/DEVC-482